mantle/kola/harness.go

```diff
@@ -340,6 +341,11 @@ type DenyListObj struct {
 	Platforms  []string `yaml:"platforms"`
 	SnoozeDate string   `yaml:"snooze"`
 	OsVersion  []string `yaml:"osversion"`
+	QemuFirmware string `yaml:"qemufirmware"`
+	Multipath    string `yaml:"multipath"`
+	Disk         string `yaml:"disk"`
+	PXE          string `yaml:"pxe"`
+	ISO          string `yaml:"iso"`
```

Comment:
umm, but for example, if I want to deny all 4k tests. How it would appear in the deny list file?  
`pattern: 4k` ? If I don't pass a full test name, let say,  it would deny the feature I added?

Reply:
We should probably agree on some field ordering in test names. Since the firmware field is one that exists in all the names, we could have it always be last. Then `pattern: *.4k.*` should always work.